securedrop_export/export.py

```diff
@@ -18,15 +18,21 @@
 ENCRYPTED_DEVICE = "encrypted_volume"
 BRLASER_DRIVER = "/usr/share/cups/drv/brlaser.drv"
 BRLASER_PPD = "/usr/share/cups/model/br7030.ppd"
-
+PCI_BUS_ID = "002:"
```

Comment:
PCI_BUS_ID for my sd-export-usb is `003:` but this changes intermittently on my system based on attach/detach events.

Reply:
I have updated this script and https://github.com/freedomofpress/securedrop-workstation/pull/305 to pass the PCI_BUS_ID from dom0 to a sd-export-* specific config file. This should now work as expected.